code_generator/End2End/GeneratedDependencies/Generated.csproj

```diff
@@ -0,0 +1,93 @@
+    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <PlatformTarget>AnyCPU</PlatformTarget>
+    <DebugType>pdbonly</DebugType>
+    <Optimize>true</Optimize>
+    <OutputPath>bin\Release\</OutputPath>
+    <DefineConstants>TRACE</DefineConstants>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+  <PropertyGroup>
+    <StartupObject>
+    </StartupObject>
+  </PropertyGroup>
+  <ItemGroup>
+    <Reference Include="Improbable.CSharp">
+      <HintPath>..\..\..\workers\unity\Library\ScriptAssemblies\Improbable.CSharp.dll</HintPath>
+    </Reference>
+    <Reference Include="Improbable.Gdk.Core">
+      <HintPath>..\..\..\workers\unity\Library\ScriptAssemblies\Improbable.Gdk.Core.dll</HintPath>
+    </Reference>
+    <Reference Include="Improbable.WorkerSdkCsharp">
+      <HintPath>..\..\..\workers\unity\Assets\Plugins\Improbable\Sdk\Improbable.WorkerSdkCsharp.dll</HintPath>
+    </Reference>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
+    <Reference Include="System.Xml.Linq" />
+    <Reference Include="System.Data.DataSetExtensions" />
+    <Reference Include="Microsoft.CSharp" />
+    <Reference Include="System.Data" />
+    <Reference Include="System.Net.Http" />
+    <Reference Include="System.Xml" />
+    <Reference Include="Unity.Entities">
+      <HintPath>..\..\..\workers\unity\Library\ScriptAssemblies\Unity.Entities.dll</HintPath>
+    </Reference>
+    <Reference Include="Unity.Entities.Hybrid">
+      <HintPath>..\..\..\workers\unity\Library\ScriptAssemblies\Unity.Entities.Hybrid.dll</HintPath>
+    </Reference>
+    <Reference Include="Unity.Mathematics">
+      <HintPath>..\..\..\workers\unity\Library\ScriptAssemblies\Unity.Mathematics.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.CoreModule">
+      <HintPath>$(UNITY_HOME)\Editor\Data\Managed\UnityEngine\UnityEngine.CoreModule.dll</HintPath>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="App.config" />
+    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Target Name="BeforeCompile">
+    <Exec Command="spatial schema generate -i ../schema -L ast_json -o ../schema_json_ast --force" />
+    <Exec Command="spatial schema generate -i ../schema -L csharp -o ./Generated_Csharp_Worker --force" />
+    <Exec Command="..\..\bin\Release\GdkCodeGenerator.exe --json-dir ..\schema_json_ast --output-dir .\Generated"/>
```

Comment:
Fine for this PR, but @jared-improbable will need to think about this for Mac compatibility?

Reply:
You can use a conditional to ensure this is only run on windows

Reply:
Let's leave this for this PR? Then @jared-improbable can make mac compatible with a condition in a mac related PR, rather than me guessing how it might work now.